pkg/cmd/pr/list/list.go

```diff
@@ -87,12 +87,12 @@ func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Comman
 				opts.Draft = strconv.FormatBool(draft)
 			}
 
-			if err := cmdutil.MutuallyExclusive(
-				"specify only `--author` or `--app`",
-				opts.Author != "",
-				opts.AppAuthor != "",
-			); err != nil {
-				return err
+			if cmd.Flags().Changed("author") && cmd.Flags().Changed("app") {
```

Comment:
The nitpickiest of the nitpick: the old condition made it legal for one of the flags to have a blank value, e.g. `--author samcoe --app ""`, whereas with the new approach that's not legal anymore. This probably only affects scripts and is probably a non-issue; just though it's worth pointing out